### PR TITLE
Change Dashboard Structure, create github links, add branch for each …

### PR DESCRIPTION
**Pull request by brkicadis**
…Shop System, improve dashboard look and add dashboard title

### CODE REVIEW

File: configuration/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+plugins:
+  - https://github.com/brkicadis/woocommerce-ee/releases
+  - https://github.com/brkicadis/shopware-ee/releases
+  - https://github.com/brkicadis/magento2-ee/releases
```
Comment (pretzelhands) [inline configuration/config.yml]: Is it intentional that these URLs point to your GitHub account? Or should it point to the Wirecard account?

File: css/style.css [hunk 2]
```diff
@@ -1,379 +1,613 @@
...
+/*    -webkit-transform: rotate(-15deg);*/
+/*    background-image: linear-gradient(to right, rgba(0,0,0,0.03), rgba(255,255,255,1), rgba(255,255,255,1));*/
+/*    width: 2300px;*/
+
+/*    position: absolute;*/
+/*    z-index: -10000;*/
+/*    left: 50%;*/
+/*    border-radius: 100px;*/
+/*    overflow-y: hidden;*/
+/*}*/
```
Comment (pretzelhands) [inline css/style.css]: Do we still need this? Since its commented out I assume it can be removed?

File: css/style.css [hunk 3]
```diff
@@ -1,379 +1,613 @@
...
+.row:after {
+    content: "";
+    display: table;
+    clear: both;
+}
+
+.card {
+    padding: 16px;
+    text-align: center;
+    /*height: 300px;*/
```
Comment (pretzelhands) [inline css/style.css]: Dead piece of code here :)

File: css/style.css [hunk 4]
```diff
@@ -1,379 +1,613 @@
...
+}
+
+.flex-grid-thirds .col {
+    width: 30%;
+    align-items: center;
+    justify-content: center;
+}
+
+.col {
+    /*border: 1px solid rgba(0, 0, 0, 0.1);*/
```
Comment (pretzelhands) [inline css/style.css]: Some more commented out styles

File: css/style.css [hunk 5]
```diff
@@ -1,379 +1,613 @@
...
+    margin: 20px 0;
+    /*border: 1px solid rgba(0, 0, 0, 0.1);*/
+    /*    background: #e53935;*/
+    /*    !*box-shadow: 0 4px 12px 0 rgba(0, 0, 0, 0.5);*!*/
+    /*    border-radius: 10px;*/
+    /*    padding: 7px;*/
+    /*    text-align: center;*/
+    /*    margin: 20px 0;*/
+    /*    !*box-shadow: inset 0 -3px #b72d2a;*!*/
+    /*    border-bottom: 3px solid #9e9e9e;*/
```
Comment (pretzelhands) [inline css/style.css]: More styles we might be able to remove.

File: controller/dashboard_two_pages.py [hunk 6]
```diff
@@ -1,290 +1,304 @@
...
+
+
+reportFile = 'report.xml'
+dashboardHtmlFile = os.path.join(os.getcwd(), 'view', 'wirecard-test-results-two-pages.html')
+dashboardSingleHtmlTestFile = os.path.join(os.getcwd(), 'view', 'wirecard-single-test-result.html')
+reportFileLink = "https://rawcdn.githack.com/wirecard/reports/master/$project/$gateway/$date/$branch/report.html"
+failTest = '<img class="FAIL" src="../images/fail.png" height="25" width="25">'
+passTest = '<img class="PASS" src="../images/success.png" height="25" width="25">'
+dateFormat = '%Y-%m-%d'
+htmlDashboardTestHeader = '''<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><title>Wirecard Gateway Test Results</title><link href="https://file.myfontastic.com/YtqivGkm3YfAnbEZneHr89/icons.css" rel="stylesheet"/><link href="../css/style.css" rel="stylesheet"/><link href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet"><link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet"></head><body><main id="main"><div id="team">Shop System Test Dashboard</div>'''
```
Comment (pretzelhands) [inline controller/dashboard_two_pages.py]: Template engine planned for future refactoring

File: controller/dashboard_two_pages.py [hunk 7]
```diff
@@ -1,290 +1,304 @@
...
+                latestDate = max(singleDate for singleDate in dateDirectory)
+                convertDateFormat = latestDate.strftime(dateFormat)
+                latestDatePath = os.path.join(searchRootDirectory, directory, subdirectory, convertDateFormat)
+                findBranches(latestDatePath)
+                for branch in findBranches(latestDatePath):
+                    reportResultFile = findLatestResultFiles(os.path.join(latestDatePath, branch))
+                    if reportResultFile:
+                        gatewayTestResults.append(process_results_file(gateway, reportResultFile))
+                        # refactorTestResults(gateway, reportResultFile, pluginName, convertDateFormat, branch)
+                        for testGateway, testResults in process_results_file(gateway, reportResultFile).iteritems():
```
Comment (pretzelhands) [inline controller/dashboard_two_pages.py]: Planned for future refactoring

File: css/style.css (hunk 2 above)
Comment (brkicadis) [reply css/style.css]: deleted.

File: controller/dashboard_two_pages.py (hunk 6 above)
Comment (brkicadis) [reply controller/dashboard_two_pages.py]: As discussed, JInja2.

**Pull request merged**